oak_functions/loader/src/server.rs

```diff
@@ -965,6 +968,19 @@ mod tests {
                 self.endpoint = Some(endpoint);
             }
         }
+
+        fn run(mut self: Box<Self>) -> tokio::task::JoinHandle<()> {
+            tokio::spawn(async move {
```

Comment:
Pass-by comment: I suggest to not have the extension do the spawning, instead just return a Future and let the caller spawn it in whatever executor it uses.

Reply:
Thank you---yes, that makes sense!